docs/comparison.md

```diff
@@ -0,0 +1,65 @@
+| Trigger Button    | [🟨][23]  | ❌                       | ❌                              | ❌                         | ❌                               | ❌                             | ❌                              | ❌                               | ❌               | ❌                                    | ❌                     | ❌                       | ❌                | ❌                                     | ✅                       | [🟨][25]                                  | ❌                     | ❌                        | ❌                | ❌                            | ❌                                 |
+| Chat Privacy      | ✅        | ❌                       | ❌                              | ❌                         | ❌                               | ❌                             | ❌                              | ❌                               | ❌               | ❌                                    | ❌                     | ❌                       | ❌                | ❌                                     | ✅                       | ❌                                        | ❌                     | ❌                        | ❌                | ❌                            | ❌                                 |
+| Non-Discord JSON  | ✅        | ❌                       | ❌                              | ❌                         | ❌                               | ❌                             | ❌                              | ❌                               | ❌               | ❌                                    | ❌                     | ✅                       | ❌                | ❌                                     | ❌                       | ❌                                        | ❌                     | ❌                        | ❌                | ❌                            | ❌                                 |
+| [Recent Bump][22] | ✅        | ❌                       | ❌                              | ✅                         | ✅                               | ✅                             | ✅                              | ❌                               | ✅               | ❌                                    | ❌                     | ✅                       | ✅                | ❌                                     | ✅                       | ❌                                        | ❌                     | ✅                        | ❌                | ❌                            | ❌                                 |
```

Comment:
I don't personally care about the "Recent Bump" row - if a plugin is supposed to do one thing, and it still works, I don't think it getting an "X" is fair